site/docs/patterns/international-phone-number-input.mdx

```diff
@@ -0,0 +1,124 @@
+
+## Pattern content
+
+- The pattern consists of two form controls: a [dropdown](../components/dropdown) that displays country selection (with a country symbol component thae user can expand or collapse), and an [input](../components/input) that displays the phone number.
+- When the user selects the country, the text input adornment in the phone number input will display the country code.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/international-phone-number-input/pattern_without_country.png",
+      alt: "Phone number input pattern without selected country",
+    },
+    {
+      src: "/img/patterns/international-phone-number-input/pattern_with_country.png",
+      alt: "Phone number input pattern with selected country",
+    },
+  ]}
+  label="Select United Kingdom"
+/>
+
+## Stack layout: column vs. row
+
+You can display the pattern in [stack layout](../components/stack-layout) as a column or a row.
+
+#### Column layout
+
+Column layout has two form controls as a vertical column. We suggest using the stack layout for better visibility of the country name and the country code.
+
+<Diagrams>
+  <Diagram
+    src="/img/patterns/international-phone-number-input/pattern_with_country_symbol.png"
+    alt="Phone number input pattern with country symbol"
+    contentPosition="top"
+    border
+  >
+    #### With country symbol
+  </Diagram>
+  <Diagram
+    src="/img/patterns/international-phone-number-input/pattern_without_country_symbol.png"
+    alt="Phone number input pattern without country symbol"
+    contentPosition="top"
+    border
+  >
+    #### Without country symbol
+  </Diagram>
+</Diagrams>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+  - We suggest using the full country name instead of the abbreviation because users may
+  filter the selection by the country name.
+</ExampleContainer>
+
+### Row layout
+
+Row layout has two form controls as a horizontal row. You can use the row layout when there's a need to save space.
+
+<Diagram
+  src="/img/patterns/international-phone-number-input/pattern_row_layout.png"
+  alt="Pattern row layout"
+  border
+/>
+
+When the country name is wider than the input, it will be truncated. In this case, we suggest using the column layout.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/international-phone-number-input/pattern_truncated_row_layout.png",
+      alt: "Truncated row layout",
+    },
+    {
+      src: "/img/patterns/international-phone-number-input/pattern_truncated_column_layout.png",
+      alt: "Column layout",
+    },
+  ]}
+  label="Column layout"
+/>
+
+## Country selection dropdown
+
+The country selection dropdown uses the [List](../components/list) component and the country list should be in the alphabetical order.
```

Comment:
```suggestion
The country selection dropdown uses the [list](../components/list) component and the country list should be in the alphabetical order.
```

@sion-docs should this be lowercase like the similar references

Reply:
other component refs were updated to caps start